doc/introduction/chemistry.rst

```diff
@@ -104,3 +112,179 @@ The circuit parameter can be optimized using the interface of choice.
+---------------------------------------
+
+PennyLane supports the following quantum chemistry functions and classes.
+
+Molecular integrals and matrices
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.attraction_integral
+    ~pennylane.qchem.dipole_integrals
+    ~pennylane.qchem.electron_integrals
+    ~pennylane.qchem.electron_repulsion
+    ~pennylane.qchem.kinetic_integral
+    ~pennylane.qchem.moment_integral
+    ~pennylane.qchem.overlap_integral
+    ~pennylane.qchem.repulsion_integral
+    ~pennylane.qchem.attraction_matrix
+    ~pennylane.qchem.core_matrix
+    ~pennylane.qchem.kinetic_matrix
+    ~pennylane.qchem.mol_density_matrix
+    ~pennylane.qchem.mol_density_matrix
+    ~pennylane.qchem.moment_matrix
+    ~pennylane.qchem.overlap_matrix
+    ~pennylane.qchem.repulsion_tensor
+    ~pennylane.qchem.contracted_norm
+    ~pennylane.qchem.primitive_norm
+    ~pennylane.qchem.expansion
+    ~pennylane.qchem.gaussian_kinetic
+    ~pennylane.qchem.gaussian_moment
+    ~pennylane.qchem.gaussian_overlap
+    ~pennylane.qchem.hermite_moment
+    ~pennylane.qchem.nuclear_attraction
+
+:html:`</div>`
+
+
+Differentiable Hartree-Fock
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.hf_energy
+    ~pennylane.qchem.nuclear_energy
+    ~pennylane.qchem.scf
+
+:html:`</div>`
+
+
+non-Differentiable Hartree-Fock
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.decompose
+    ~pennylane.qchem.meanfield
+    ~pennylane.qchem.one_particle
+    ~pennylane.qchem.two_particle
+
+
+:html:`</div>`
+
+
+Differentiable observables
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.diff_hamiltonian
+    ~pennylane.qchem.dipole_moment
+    ~pennylane.qchem.fermionic_dipole
+    ~pennylane.qchem.fermionic_hamiltonian
+    ~pennylane.qchem.fermionic_observable
+    ~pennylane.qchem.jordan_wigner
+    ~pennylane.qchem.molecular_hamiltonian
+    ~pennylane.qchem.observable
+    ~pennylane.qchem.qubit_observable
+    ~pennylane.qchem.simplify
+
+
+:html:`</div>`
+
+
+non-Differentiable observables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.dipole_of
+    ~pennylane.qchem.molecular_hamiltonian
+
+
+:html:`</div>`
+
+Other observables
+^^^^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.particle_number
+    ~pennylane.qchem.spin2
+    ~pennylane.qchem.spinz
+
+:html:`</div>`
+
+Qubit tapering
+^^^^^^^^^^^^^^
+
+:html:`<div class="summary-table">`
+
+.. autosummary::
+    :nosignatures:
+
+    ~pennylane.qchem.clifford
+    ~pennylane.qchem.optimal_sector
```

Comment:
Shouldn't `optimal_sector` be also made available outside `qchem`? 🤔

Reply:
It requires number of electrons which makes it qchem-specific.

Reply:
out of curiosity, is there a way of making this functionality non-chemistry specific?

Reply:
One way is to modify the function to _optionally_ perform a brute-force check and return the sector with minimum energy. But I am not sure if there is a general way to find the optimal sector for any given Hamiltonian.
@obliviateandsurrender might have a better insight.